packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -49,17 +57,60 @@ export default function DocumentActions( { documentTitle } ) {
 		>
 			{ documentTitle ? (
 				<>
-					<div
-						className={ classnames(
-							'edit-site-document-actions__label',
-							'edit-site-document-actions__title',
+					<Dropdown
+						position="bottom center"
+						renderToggle={ ( { onToggle, isOpen } ) => {
+							const openOnArrowDown = ( event ) => {
+								if ( ! isOpen && event.keyCode === DOWN ) {
+									event.preventDefault();
+									event.stopPropagation();
+									onToggle();
+								}
+							};
 							{
-								'is-active': isTitleActive,
+								/* TODO: Fix vertical text padding */
 							}
+							return (
+								<Button
+									onClick={ onToggle }
+									className={ classnames(
+										'edit-site-document-actions__label',
+										'edit-site-document-actions__title',
+										{
+											'is-active': isTitleActive,
+										}
+									) }
+									aria-haspopup="true"
+									aria-expanded={ isOpen }
+									onKeyDown={ openOnArrowDown }
+									label={ __( 'Change document settings.' ) }
+									showTooltip
+								>
+									{ documentTitle }
+								</Button>
+							);
+						} }
+						renderContent={ () => (
+							<div
```

Comment:
I think it would be helpful to split out the actual settings area into a separate component, even at this early stage :) (including the page selector above)

Reply:
Totally agreed 👍